script.js

```diff
@@ -2,21 +2,181 @@ const hamburger = document.querySelector('.bars');
+      full_desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+      image: './images/popup.png',
+      tags: ['html', 'bootstrap', 'ruby'],
+    },
+    {
+      id: 5,
+      title: 'Profesional Art Printing Data-5',
+      short_desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      full_desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+      image: './images/popup.png',
+      tags: ['html', 'bootstrap', 'ruby'],
+    },
+    {
+      id: 6,
+      title: 'Profesional Art Printing Data-6',
+      short_desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      full_desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+      image: './images/popup.png',
+      tags: ['html', 'bootstrap', 'ruby'],
+    },
+  ],
+};
+
+const featuredHtml = `<div class="feature">
+  <img src=${projectCardList.featured.image} class="yoga-image" alt="">
+  <div class="project-detail">
+    <h3 class="project-title">${projectCardList.featured.title}</h3>
+    <p class="project-text">${projectCardList.featured.short_desc}</p>
+    <ul class="languages">
+    ${projectCardList.featured.tags
+    .map((tagName) => `<li class="skills">${tagName}</li>`)
+    .join('')}
+    </ul>
+    <button class="see-project-button" type="button">See Project</button>
+  </div>
+</div>`;
+
+workS.appendChild(
+  ...ConvertStringToHTML(featuredHtml).children,
+);
+
+const projectUnorderedList = document.createElement('ul');
+projectUnorderedList.className = 'random-projects';
```

Comment:
- [ ] When the user hover over the random-projects list, all of the projects are getting larger and part of them are going out of the screen and overlapping other content, creating a bad user experience. Please fix that.
Your design:
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/61919617/181068481-792430c0-42c5-4e36-9274-e94bb471ee86.png">